content/about/releases.md

```diff
@@ -9,16 +9,30 @@ tags: ["Release"]
 
 ## License
 
-Kokkos and Kokkos-Kernels are distributed under an Apache 2.0 with LLVM exception license, an [OSI-approved](https://opensource.org/licenses/Apache-2.0) license.
-The license files are available here: [kokkos](https://raw.githubusercontent.com/kokkos/kokkos/develop/LICENSE), [kokkos-kernels](https://raw.githubusercontent.com/kokkos/kokkos-kernels/develop/LICENSE).
+Currently, Kokkos and Kokkos-Kernels are distributed under the [Apache 2.0 with
+LLVM exception license](
+https://raw.githubusercontent.com/kokkos/kokkos/refs/tags/4.0.00/LICENSE),
+an [OSI-approved](https://opensource.org/licenses/Apache-2.0) license.
+
+Releases prior to 4.0 were licensed under the [3-clause BSD license](
+https://raw.githubusercontent.com/kokkos/kokkos/refs/tags/3.0.00/LICENSE).
+
 
 ## GnuPG signing keys
 
-The maintainers of the libraries sign the tags associated with releases and sign the source packages. In order to verify the authenticity of the source packages you can download the public keys here: [dalg24.asc](content/siging-keys/release-keys.asc) and [lucbv.asc](https://kokkos.org/downloads/signing-keys/lucbv.asc).
+Security is important to us.  To ensure the integrity and authenticity of the
+source packages for our releases, we digitally sign all release tags and
+source code archives.  You can verify these signatures using our public keys,
+available for download here:
+[dalg24.asc](content/siging-keys/release-keys.asc) and
+[lucbv.asc](https://kokkos.org/downloads/signing-keys/lucbv.asc).
+
 
 ## Download
 
 | Date | Version | Download | Release Notes |
```

Comment:
Remove the `Release Notes` column or populate it. Right now, we have an empty column

Reply:
I was not sure what to do with these.  We have a change log but not really release notes.
But sure I will remove.